### PR TITLE
add device_token to oauth payload to reenable oauth

### PR DESCRIPTION
**Pull request by fkotsian**
hey @rghuckins ! Thanks for this awesome lib 😄  Saw a small missing attr from the oauth payload and thought I would submit a PR to add back in

Happy gambooling 😁

### CODE REVIEW

**Comment by rghuckins** [comment]
Thanks @fkotsian!

**Comment by ParitoshKelkar** [comment]
Hey @fkotsian just checking to see if the additional payload should be **device_token** instead of **device_id**. The patch on the PR adds **device_id** but I had to change the field to **device_token** for the call to work .. (Was running into issue #9 )
Not sure if that was just for me though. 
Thanks @rghuckins, @fkotsian for your efforts on this tool!

File: robinhood.gs [hunk 1]
```diff
@@ -33,7 +33,8 @@ function getAccessToken_() {
     'scope': "internal",
     'client_id': "c82SH0WZOsabOXGP2sxqcj34FxkvfnWRZBKlBjFS",
     'username': robinhoodUsername,
-    'password': robinhoodPassword
+    'password': robinhoodPassword,
+    'device_id': "<grab via login to Web>"  /* https://stackoverflow.com/questions/55902967/is-there-any-way-to-get-a-bearer-token-now-since-robinhood-has-changed-the-api/56841942 */
```
Comment (fkotsian) [inline robinhood.gs:37]: @ParitoshKelkar You're correct, the cookie is called `device_id` in the web inspector, and should be `device_token` here

Use device_token for this field guys!

**Comment by fkotsian** [comment]
I think this is good to merge!

**Comment by rghuckins** [comment]
Thanks again @fkotsian 👍

**Pull request merged**